demo/MenuViewController.swift

```diff
@@ -167,17 +180,17 @@ final class MenuViewController: UIViewController {
     }
     
     @objc private func infoTapped() {
-        let alert = UIAlertController(title: L10n.infoDialog, message: nil, preferredStyle: .actionSheet)
+        let alert = UIAlertController(title: L10n.infoTitle, message: nil, preferredStyle: .actionSheet)
 
-        alert.addAction(UIAlertAction(title: L10n.termsOfServiceOption, style: .default, handler: { _ in
-            guard let url = URL(string: "https://www.mapbox.com/tos/") else { return }
+        alert.addAction(UIAlertAction(title: L10n.generalTermsOfService, style: .default, handler: { [weak self] _ in
```

Comment:
`[weak self]` would be unused in case of utilizing global constant for the links.

Reply:
done